src/index.ts

```diff
@@ -1,40 +1,18 @@
-import AsyncAirtable = require("asyncairtable");
-import { AirtableRecord, QueryObject } from "asyncairtable/lib/@types";
-
-const asyncAirtable = new AsyncAirtable("keyyCSADD61J0D4wQ", "appGzEQkD3QySIDvj", { baseURL: "https://api.vacfind.org/v0" });//, { ...CONFIG }
```

Comment:
This is totally my fault for pointing you to the wrong documentation, but the baseURL config item was recently added in `async-airtable` version 2.2 (`next`,i believe). This allows us to use our API to cache requests to airtable so we are less likely to go over their ratelimits. You may need to build the documentation from the repo (https://github.com/GV14982/async-airtable) to get the latest version until they make a new release that contains this change.